tests/Service/BadgesTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Service;
+
+use App\Service\Badges;
+use PHPUnit\Framework\TestCase;
+
+final class BadgesTest extends TestCase
+{
+    public function testGetAllBadges(): void
+    {
+        $expected = [
+            [
+                'name' => 'latest_stable_version',
+                'label' => 'Latest Stable Version',
+                'route' => 'pugx_badge_version_latest',
+                'latest' => 'stable',
+            ],
+            [
+                'name' => 'total',
+                'label' => 'Total Downloads',
+                'route' => 'pugx_badge_download',
+            ],
+            [
+                'name' => 'latest_unstable_version',
+                'label' => 'Latest Unstable Version',
+                'route' => 'pugx_badge_version_latest',
+                'latest' => 'unstable',
+            ],
+            [
+                'name' => 'license',
+                'label' => 'License',
+                'route' => 'pugx_badge_license',
+            ],
+            [
+                'name' => 'monthly',
+                'label' => 'Monthly Downloads',
+                'route' => 'pugx_badge_download_type',
+                'type' => 'monthly',
+            ],
+            [
+                'name' => 'daily',
+                'label' => 'Daily Downloads',
+                'route' => 'pugx_badge_download_type',
+                'type' => 'daily',
+            ],
+            [
+                'name' => 'version',
+                'label' => 'Version',
+                'route' => 'pugx_badge_version',
+            ],
+            [
+                'name' => 'require_php',
+                'label' => 'PHP Version Require',
+                'route' => 'pugx_badge_require',
+                'type' => 'php',
+            ],
+            [
+                'name' => 'composerlock',
+                'label' => 'composer.lock',
+                'route' => 'pugx_badge_composerlock',
+            ],
+            [
+                'name' => 'gitattributes',
+                'label' => '.gitattributes',
+                'route' => 'pugx_badge_gitattributes',
+            ],
+            [
+                'name' => 'dependents',
+                'label' => 'Dependents',
+                'route' => 'pugx_badge_dependents',
+            ],
+            [
+                'name' => 'suggesters',
+                'label' => 'Suggesters',
+                'route' => 'pugx_badge_suggesters',
+            ],
+            [
+                'name' => 'circleci',
+                'label' => 'CircleCI Build',
+                'route' => 'pugx_badge_circleci',
+            ],
+        ];
+
+        $this->assertEquals($expected, Badges::getAll());
```

Comment:
This looks a bit too much verbose to me... maybe we can just limit it to assert that the returned value is an array?

Reply:
Yes, it's verbose but in my opinion, it's better than checking if the return value is an array, so if you change the dictionary, there is a test to validate it.
What do you think?